code/game/gamemodes/clock_cult/clock_structures.dm

```diff
@@ -293,9 +293,7 @@
 		user.visible_message("<span class='notice'>[user] places [S] in [src], where it fuses to the shell.</span>", "<span class='brass'>You place [S] in [src], fusing it to the shell.</span>")
 		var/mob/living/simple_animal/A = new mobtype(get_turf(src))
 		A.visible_message("<span class='brass'>[src][spawn_message]</span>")
-		remove_servant_of_ratvar(S.brainmob, TRUE)
```

Comment:
This was to get around body-based stuff for mind-based antags, which this fixes.